rds-deployment-ha-gateway/azuredeploy.json

```diff
@@ -8,16 +8,23 @@
         "description": "Unique public DNS prefix for the deployment. The fqdn will look something like '<dnsname>.westus.cloudapp.azure.com'. Up to 62 chars, digits or dashes, lowercase, should start with a letter: must conform to '^[a-z][a-z0-9-]{1,61}[a-z0-9]$'."
       }
     },
-    "loadBalancer": {
+    "gatewayloadBalancer": {
```

Comment:
why can't we keep "loadBalancer" as the parameter name? remind me, is there another load balancer in the deployment?

if you're chose to keep it as "gatewayloadBalancer" follow camelcase: "gatewayLoadBalancer"